### DISCUSSION TITLE
An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help

### DISCUSSION THREAD

**Discussion by Vette1123** [vercel/community#6649] [Help]
### Summary

<html>
<body>

An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help
11:26:26.237

cannot deploy my project

![image](https://github.com/vercel/community/assets/26345237/77896d0a-c7df-47be-a5d9-53c4f54cfefc)

<br class="Apple-interchange-newline">
</body>
</html>

### Steps to Reproduce

<html>
<body>

An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help
11:26:26.237

cannot deploy my project

![image](https://github.com/vercel/community/assets/26345237/77896d0a-c7df-47be-a5d9-53c4f54cfefc)

<br class="Apple-interchange-newline">
</body>
</html>

**Comment by eoinoconnell** [comment]
Happening to me also

**Comment by Vette1123** [comment]
Any updates guys?, i still get this error!! 

@leerob

**Reply by eoinoconnell** [reply]
My issue was due to a pinned version of the vercel cli in my environment variables if that helps. I removed the env var and back working now
